### PR TITLE
added --preserve-map-align-order

### PR DESCRIPTION
**Pull request by ekiernan**
Adding a fix for DRAGEN non-determinism back into the DragenAlignment WDL. Scientific tests in comments

### CODE REVIEW

**Comment by ekiernan** [comment]
WGS scientific test: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/84289/

File: pipelines/broad/reprocessing/external/wgs/ExternalWholeGenomeReprocessing.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2.3.1
+2024-09-17 (Date of Last Commit)
+
+* Updated DRAGEN alginer parameters to fix non-determinism
```
Comment (nikellepetrillo) [inline pipelines/broad/reprocessing/external/wgs/ExternalWholeGenomeReprocessing.changelog.md:4]:
```suggestion
* Updated DRAGEN aligner parameters to fix non-determinism
```

File: pipelines/broad/reprocessing/wgs/WholeGenomeReprocessing.changelog.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 3.3.1
+2024-09-17 (Date of Last Commit)
+
+* Updated DRAGEN alginer parameters to fix non-determinism
```
Comment (nikellepetrillo) [inline pipelines/broad/reprocessing/wgs/WholeGenomeReprocessing.changelog.md:4]:
```suggestion
* Updated DRAGEN aligner parameters to fix non-determinism
```

Comment (ekiernan) [reply pipelines/broad/reprocessing/wgs/WholeGenomeReprocessing.changelog.md]: Thank you!

**Review by kevinpalis — APPROVED** [review]

**Pull request merged**